frontend/.agents/skills/resend/references/installation.md

```
@@ -0,0 +1,142 @@
+| `mix.exs` | Elixir | resend |
+
+## Installation Commands
+
+### Node.js
+
+```bash
+npm install resend
+```
+
+Alternative package managers:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)